recipe/meta.yaml

```diff
@@ -65,7 +65,6 @@ about:
 extra:
   recipe-maintainers:
     - eugeneburger
-    - josborne-noaa
```

Comment:
Did you communicate this with @josborne-noaa?

Reply:
The linter complains otherwise and says this user does not exist.

Reply:
Oh! I guess he left NOAA. @eugeneburger if there is a new PyFerret developer we can add them here in the future.